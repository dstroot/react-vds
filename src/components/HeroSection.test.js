import React from 'react';
import HeroSection from './HeroSection';
import renderer from 'react-test-renderer';

test('HeroSection renders some content', () => {
  const component = renderer.create(<HeroSection />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
