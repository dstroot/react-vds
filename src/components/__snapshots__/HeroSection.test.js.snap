// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroSection renders some content 1`] = `
<div
  className="hero text-white"
  style={
    Object {
      "backgroundImage": "url(startup-3.jpg)",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="row align-items-center p-2"
    >
      <div
        className="col-md-6 offset-md-6 p-lg-5 my-5"
      >
        <h6
          className="text-uppercase bd-text-purple-bright"
        >
          High Praise
        </h6>
        <h2
          className="font-weight-light"
        >
          “Go Analytics is amazing. Decisions that used to take weeks, now only takes minutes and is available to everyone on my team.”
        </h2>
        <p
          className="font-weight-light"
        >
          Cindy Smith, founder of Cool Startup
        </p>
      </div>
    </div>
  </div>
</div>
`;
